readthedocs/config/config.py

```diff
@@ -1257,18 +1279,15 @@ def validate_keys(self):
         This should be called after all the validations are done and all keys
         are popped from `self._raw_config`.
         """
-        msg = (
-            'Invalid configuration option: {}. '
-            'Make sure the key name is correct.'
-        )
+        msg = "Make sure the key name is correct."
```

Comment:
Does this need to be a variable at all?

Reply:
Not really. I think it made sense before because it was a multi-line string.